cmd/todo_mr.go

```diff
@@ -0,0 +1,59 @@
+package cmd
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var todoMRCmd = &cobra.Command{
+	Use:              "mr",
+	Aliases:          []string{"issue"},
```

Comment:
I know it's a "draft" PR, but... hahah..
I don't think that adding one single command to handle both `mr` and `issue` is a good idea :D.
The `lab todo add` approach with flags for defining issue vs mr seems a far better option IMHO.

Reply:
Haha :)  It is definitely a work in progress:

`lab todo add 1234 --mr` : but this is kind of clunky, and that's why I came up with 

`lab todo mr 1234`  which seems like a better idea.

What about

`lab mr add 1234` ? :)  Would that be better?  Or too confusing?

Reply:
The more I think about this, the more I like the idea of having 

`lab todo add 1234 --mr`

*AND*

`lab mr add 1234`

The second command could be an easy wrapper around the first so there's a minimum of code duplication.

Reply:
`lab mr add` is too similar to `lab mr create`, isn't it?
What about `lab todo mr <mrid>` and `lab todo issue <issue-id>` ?

Reply:
Yeah, that'd work too.  Let me change the code...

Reply:
Let me know what you think of the recent push.